docker/node/Dockerfile

```diff
@@ -15,6 +10,15 @@ RUN /tmp/prepare/prepare.sh
 COPY ./ /tmp/src
 WORKDIR /tmp/src
 
+#Define ARGs for ci/build-node.sh
```

Comment:
The reason I moved these ARGs further down is so that we don't have to rerun the prepare.sh step when we change the build flags. This way we can make use of the docker cache for unchanged steps. While otherwise one changed ARG willinvalidate all next steps

Reply:
That's a very good reason, didn't think about that 👍